src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -24,7 +24,7 @@
     /**
      * Handle the / endpoint
      *
-     * @return
+     * @return index
```

Comment:
As long as you're editing @return, give a description of what it is returning.  What can a caller expect when invoking this function?